wagtail_content_import/pickers/microsoft/wagtail_hooks.py

```diff
@@ -2,9 +2,16 @@
 
 import requests
 from django.conf import settings
-from wagtail.core import hooks
+from wagtail import VERSION as WAGTAIL_VERSION
 
-from ...utils import create_page_from_import, is_importing, set_importing, update_page_from_import
+if WAGTAIL_VERSION >= (3, 0):
+    from wagtail import hooks
+else:
+    from wagtail.core import hooks
+
+from ...utils import (
+    create_page_from_import, is_importing, set_importing,
+    update_page_from_import)
```

Comment:
nitpick: this is odd formatting. However, since the package doesn't lint with black, it is one for a follow-up PR

Reply:
Hi @zerolab ,
I made a follow up PR here, specifically for including black in the lint job:
https://github.com/torchbox/wagtail-content-import/pull/51/

Thank you for this.